tests/x11/systemsettings5.pm

```diff
@@ -0,0 +1,28 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "x11test";
+use testapi;
+
+sub run() {
+    my $self = shift;
+    x11_start_program("systemsettings5", 6, {valid => 1});
+    if (get_var("LIVETEST")) {
+        assert_screen 'test-systemsettings-1', 15;
```

Comment:
well, as I said above, I don't mind to use default timeout. and I don't know what is the current agreement/rule to treat timeout with assert_screen and check_screen.